srai/embedders/gtfs2vec_embedder.py

```diff
@@ -0,0 +1,271 @@
+"""
+gtfs2vec embedder.
+
+This module contains embedder from gtfs2vec paper [1].
+
+References:
+    [1] https://doi.org/10.1145/3486640.3491392
+"""
+
+
+from functools import reduce
+from typing import Any, Dict, List, Optional
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+from torch.utils.data import DataLoader
+
+from srai.embedders import BaseEmbedder
+from srai.models import GTFS2VecModel
+from srai.utils.constants import GTFS2VEC_DIRECTIONS_PREFIX, GTFS2VEC_TRIPS_PREFIX
+from srai.utils.exceptions import ModelNotFitException
+
+
+class GTFS2VecEmbedder(BaseEmbedder):
+    """GTFS2Vec Embedder."""
+
+    def __init__(
+        self, hidden_size: int = 48, embedding_size: int = 64, skip_embedding: bool = False
```

Comment:
Can you think of another name you could use for `skip_embedding`? I think this might be a bit confusing to instantiate an `Embedder` that `skips_embedding`, though I'm not sure what to call it.

Reply:
`skip_autoencoder`?